### PR TITLE
storage: use static metrics for raw read

### PR DESCRIPTION
**Pull request by fredchenbj**
Signed-off-by: fredchenbj <cfworking@163.com>

## What have you changed? (mandatory)
Change the global metics to static metrics in raw read.

## What are the type of the changes? (mandatory)

The currently defined types are listed below, please pick one of the types for this PR by removing the others:
- Improvement (change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)
Manual tests.

## Does this PR affect documentation (docs) update? (mandatory)
No.

## Does this PR affect tidb-ansible update? (mandatory)
no.

### CODE REVIEW

File: src/storage/metrics.rs [hunk 1]
```diff
@@ -43,6 +57,16 @@ lazy_static! {
         exponential_buckets(0.0005, 2.0, 20).unwrap()
     )
     .unwrap();
+    pub static ref SCHED_RAW_READ_HISTOGRAM_VEC: SchedDurationVec = {
+        register_static_histogram_vec!(
+            SchedDurationVec,
+            "tikv_scheduler_raw_read_command_duration_seconds",
```
Comment (siddontang) [inline src/storage/metrics.rs]: em, do you introduce another new metric name here? If yes, please use the old one.

/cc @breeswish

**Comment by siddontang** [comment]
how about the performance? I remember that before when we used static metrics, the QPS performance increased 7%.

**Comment by fredchenbj** [comment]
@siddontang In my testing case, the QPS (12914->13163) increased 2%, average latency (1207->1175) decreased 2.7%, and p999 latency (24655->4835) decreased obviously.

File: src/storage/metrics.rs (hunk 1 above)
Comment (fredchenbj) [reply src/storage/metrics.rs]: here, I will change to use the old one.

**Comment by siddontang** [comment]
Cool @fredchenbj

**Comment by siddontang** [comment]
Maybe we can refactor KV_COMMAND_COUNTER_VEC later @fredchenbj

File: src/storage/mod.rs [hunk 2]
```diff
@@ -1109,9 +1109,7 @@ impl<E: Engine> Storage<E> {
         let engine = self.get_engine();
         let priority = readpool::Priority::from(ctx.get_priority());
 
-        let timer = SCHED_HISTOGRAM_VEC
-            .with_label_values(&[CMD])
-            .start_coarse_timer();
+        let timer = SCHED_RAW_READ_HISTOGRAM_VEC.raw_get.start_coarse_timer();
```
Comment (AndreMouche) [inline src/storage/mod.rs]: Do not forget to change the metrics to the old one.

**Comment by fredchenbj** [comment]
@siddontang Do I need refactor KV_COMMAND_COUNTER_VEC in this pr ?

**Comment by siddontang** [comment]
@fredchenbj 

No, you don't need to do in this PR, you can create another PR do it :-)

**Pull request merged**